pkg/annotation/aws.go

```diff
@@ -1,41 +1,86 @@
 package annotation
 
-// AWSCNIMinimumIPTarget is an annotation to configure the value for MINIMUM_IP_TARGET for AWS CNI
-// see https://github.com/aws/amazon-vpc-cni-k8s#cni-configuration-variables
-// and https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/eni-and-ip-target.md
+//Keep the YAML documentation format as it is used to render in the CRD public documentation. You can use Markdown in the documentation field.
+
+// support:
+//   - crd: awsclusters.infrastructure.giantswarm.io
+//     apiversion: v1alpha2
+//     release: "Since 14.0.0"
```

Comment:
```suggestion
//     release: Since 14.0.0
```

These quotes shouldn't be needed in YAML